common/js/src/i18n.js

```diff
@@ -74,27 +75,52 @@ function transformAllElements(attribute, transformFunction) {
   }
 }
 
+function transformFunctionI18nDataAttribute(element, translatedText) {
+  logger.fine('Translating element.textContent [' + element.outerHTML +
+              '] to "' + translatedText + '"');
+  element.textContent = translatedText;
+}
+
+function transformFunctionI18nDataAriaLabelAttribute(element, translatedText) {
+  logger.fine('Translating element.aria-label [' + element.outerHTML +
+              '] to "' + translatedText + '"');
+  element.setAttribute('aria-label', translatedText);
+}
+
+/** 
+ * Takes the single element passed, replacing element.textContent with translation
+ * if it conatins I18N_DATA_ATTRIBUTE, and setting aria-label to translation
+ * if it contains I18N_DATA_ARIA_LABEL_ATTRIBUTE.
+*/
+GSC.I18n.adjustElementTranslation = function(element) {
+  if (goog.dom.dataset.has(element, 'i18n')) {
```

Comment:
nit: I'd suggest using `.hasAttribute(I18N_DATA_ATTRIBUTE)` instead, since not using the constant here kinda defeats its purpose.

Ditto in line 102.